packages/traceability-schemas/schemas/Event.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/Event.json",
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"Event\", \"@id\": \"https://schema.org/Event\"}",
+    "title": "Organization",
+    "description": "An event such as transformation, aggregation, commission etc.",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "type": "array"
+        },
+        "type": {
+            "oneOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array"
+                }
+            ]
+        },
+        "eventType": {
+            "$comment": "{\"term\": \"eventType\", \"@id\": \"https://schema.org/value\"}",
+            "title": "Event Type",
+            "description": "Type of Event",
+            "type": "string"
+        },
+        "eventId": {
+            "$comment": "{\"term\": \"eventId\", \"@id\": \"https://schema.org/identifier\"}",
+            "title": "Event Identifier",
+            "description": "Event Identifier.",
+            "type": "string"
+        },
+        "actor": {
+            "$comment": "{\"term\": \"actor\", \"@id\": \"https://w3id.org/traceability#Organization\", \"@type\": \"https://schema.org/ItemList\"}",
+            "title": "Actor",
+            "description": "The organization performing the activity.",
+            "type": "array",
+            "items": {
+                "$ref": "https://w3id.org/traceability/schemas/Organization.json"
+            }
+        },
+        "eventLocation": {
+            "$comment": "{\"term\": \"eventLocation\", \"@id\": \"https://w3id.org/traceability#eventLocation\"}",
+            "title": "Event Location",
+            "description": "Location where event took place",
+            "$ref": "https://w3id.org/traceability/schemas/Place.json"
+        },
+        "eventTime": {
+            "$comment": "{\"term\": \"eventTime\", \"@id\": \"https://schema.org/DateTime#v2\"}",
+            "title": "Event Time",
+            "description": "Time when the event took place",
+            "type": "string"
+        },
+        "products": {
```

Comment:
I am going to have to mull over use of Product here.  I think it is probably correct, but also thinking about raw materials becoming a product etc.

Reply:
see note from @OR13 below - should this be GS1 or schema.org?  Other aspects of the vocab are utilizing schema.org

Reply:
Consider "inputs" (raw materials, outputs from other processes/sources, etc.) and "outputs" (refined version of inputs becoming inputs to other processes/consumers, finished products, etc.).  These can then be easily turned into a complex *trace* from most-basic-origin (e.g., extraction of various ores from deep underground) to most-refined-consumable (e.g., a smartphone, tablet, or computer)...